keyboards/hineybush/sm68/readme.md

```diff
@@ -0,0 +1,15 @@
+# sm68
+
+![sm68](imgur.com image replace me!)
```

Comment:
Please put an image here :)

Reply:
waiting for pics of the board, which are TBA

Reply:
In the meanwhile, could you put a note here instead?  
I mean, unless the image is a couple of days out.